components/PastClients.jsx

```diff
@@ -0,0 +1,12 @@
+export default function PastClients({ pastClients }) {
+  return (
+    <section aria-labelledby="Skills-Header" className="cmp-past-clients">
+      <h2 id="Skills-Header" className="cmp-past-clients__heading">Past Clients</h2>
```

Comment:
Lets update these ID's to be uniqu
![Screen Shot 2022-05-20 at 8 49 33 AM](https://user-images.githubusercontent.com/49104801/169554262-ce4da730-5b6b-43df-a69c-5294da87328f.png)
e

Reply:
When copy and past goes wrong